docs/source/user-guide/configuration/use-condarc.rst

```diff
@@ -691,6 +692,24 @@ EXAMPLE:
    This is forced to ``True`` if conda-build is installed and older than 3.18.3,
    because older versions of conda break when conda feeds it the new file format.
 
+.. _set-default-env:
+Specify a default environment to activate on launch
+---------------------------------------------------
+It is now possible to set a default conda environment at launch, so when you open 
+an Anaconda Prompt, or enter ``conda activate`` at the command line, it will activate 
+your preferred environment rather than ``base``. This is done by setting the 
+``default_start_environment`` variable.
```

Comment:
```suggestion
Conda 23.5.0 introduced the ability to set a default environment on shell start-up. This is done
by setting the ``default_start_environment`` variable.
```

Reply:
@electric-souperman, we now avoid referencing Anaconda within the conda documentation. Additionally, our users can use more than just the Anaconda prompt to run conda commands, so the above may be confusing to those users.